src/jekyll/_plugins/custom_tags/audio.rb

```diff
@@ -0,0 +1,59 @@
+# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
*fist bump*

Generally the YouTube got out of hand and an include is to easy for someone
to jump in an alter, also include may be missing something and will still
print nothing, tags give easier error checking and throwing.
On Tue, 20 Oct 2015 at 15:54 Pete LePage <notifications@github.com> wrote:

> In src/jekyll/_plugins/custom_tags/audio.rb
> <https://github.com/google/WebFundamentals/pull/2179#discussion_r42505976>
> :
>
> > @@ -0,0 +1,59 @@
> > +# Copyright 2014 Google Inc. All rights reserved.
>
> [image: :facepunch:] - nah, that's too much work. if it's his, then we'll
> go with that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/WebFundamentals/pull/2179/files#r42505976>.
>